src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -183,10 +180,22 @@ private static Optional<LfAction> create(LoadAction action, LfNetwork lfNetwork,
     }
 
     private static Optional<LfAction> create(PhaseTapChangerTapPositionAction action, LfNetwork lfNetwork) {
-        LfBranch branch = lfNetwork.getBranchById(action.getTransformerId()); // only two windings transformer for the moment.
+
+        String legNumber = "";
+
+        Optional<ThreeWindingsTransformer.Side> side = action.getSide();
+        if (side.isPresent()) {
```

Comment:
You already have this need for `PropagatedContingency`, so we have to refactor to share code. Note that the initialization of leg number with `""` is not really nice. Better put something like:
````
 LfBranch branch = lfNetwork.getBranchById(side.isPresent() ? action.getTransformerId() + legNumber : action.getTransformerId());
````